linkbelt/src/services/config.test.ts

```diff
@@ -0,0 +1,126 @@
+import mock from 'mock-fs'
```

Comment:
Should this be in the test directory?

Reply:
I try to keep more unit-ish tests beside the file, and large functional/integration tests outside of the src dir

Reply:
"unit-ish" sounds kind of vague, and this stands out as an outlier atm. Do we do this commonly in other packages? I'd prefer picking a place to look, so people can know where to look when updating and running tests before finding out on CI.

Reply:
We don't really have a standard test guideline.
- Explorer has all tests in `src/__tests__` but it doesnt separate unit and integration tests.
- Explorer-client has unit tests in `src/__tests__` and integration tests in `/integration`
- Operator-ui has unit tests in `__tests__` (not in `src/`) and integration tests in `/integration`
- `@chainlink/contracts` has tests in `tests`

I try to put tests that are unit tests (Not sure why I called this one unit-ish, it's definitely a unit test. I forgot that I mock out the filesystem for this test too.) right beside their files for easily discoverability, and functional/integration tests at top of the package. Happy to change if you think they should be in the top level `tests` folder

Reply:
All the parts that use `integration` test across multiple parts of the repo; `core/`, `operator-ui/`, `explorer`... I think the term "integration" is vague, so if it's within one part of the repo, I lean towards "feature" test. I'm not as familiar with the patterns within the TypeScript packages, but at first glance it still makes sense for me to group them together. Your call.

Reply:
I'd vouch for us keeping it as-is and pushing for a guideline for testing in the future to abide by